docs/content/docs/development/create-table.md

```diff
@@ -268,3 +268,45 @@ For example, the inputs:
 
 Output: 
 - <1, 25.2, 20, 'This is a book'>
+
+## Aggregation Update
+
+You can configure partial update from options:
+
+```sql
+CREATE TABLE MyTable (
+  a STRING,
+  b INT,
+  c INT,
+  PRIMARY KEY (a) NOT ENFORCED 
+) WITH (
+      'merge-engine' = 'aggregation',
+      'b.aggregate-function' = 'sum',
+      'c.aggregate-function' = 'sum'
+);
+```
+{{< hint info >}}
+__Note:__Aggregate updates are only supported for tables with primary keys.
+{{< /hint >}}
+
+{{< hint info >}}
+__Note:__Aggregate updates do not support streaming consumption.
```

Comment:
Add a test case for streaming consumption. This test should throw an exception telling the user that we don't support this.